frontend/app/utils/date-utils.ts

```diff
@@ -213,3 +213,15 @@ export function useMonths(locale: string, format: 'numeric' | '2-digit' | 'long'
     });
   }, [format, locale]);
 }
+
+/**
+ * Custom function that returns the current date formatted ex.: 2020-12-30.
+ * @returns the current date as a string.
+ */
+export function getCurrentDate(): string {
+  const today = new Date();
+  const year = today.getFullYear();
+  const month = String(today.getMonth() + 1).padStart(2, '0'); // Months are 0-based
+  const day = String(today.getDate()).padStart(2, '0');
+  return `${year}-${month}-${day}`;
+}
```

Comment:
Use `UTCDate` to get date object.

```suggestion
export function getCurrentDateString(locale: AppLocale = 'en'): string {
  return new UTCDate().toLocaleDateString(`${locale}-CA`);
}
```